rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -2176,6 +2171,33 @@ private <T> T pollQueue() {
         }
     }
 
+    private void getKeyword(String variableName, StringBuilder keyword) {
```

Comment:
I don't think I should use a StringBuilder here, but just return a String. The case someone uses multiple keywords is rather small, so for like > 95% of all scripts this function will only be run once per variable. So the performance gain is really small.

Reply:
That part seemed odd to me as well; clearer to just return a String indeed.